api/src/main/resources/fireCloud.yaml

```diff
@@ -290,7 +397,7 @@ definitions:
           description: User's title
         contactEmail:
           type: string
-          description: User's contact email
+          description: User's contact email (if different from account email)
```

Comment:
If the workspace does just use the account email as the contact email, will this return as null or as the account email?

Reply:
I don't know; this is copied from the FC Swagger spec. @dmohs may know.

Reply:
I talked to @dmohs and he mentioned that it isn't guaranteed. (That is why the contact email is an optional field, because users might just use the same email for login and contact.)

Do we want to ensure existence of a contact email in our DB, or should I create a follow up PR to this one to check if there is no contact email, and if there isn't one, then send bug reports from the `email` field?

Reply:
So given that 1) we require users to use researchallofus.org@ accounts when accessing the workbench, 2) we want a way to contact participants, I would say we should make contact email required... and it shouldn't really default to the Google account they signed in with because that will go nowhere.

In any case, feels separate from this PR.

Reply:
Note: we would only make it required once we actually have the registration flow to populate it. :)